src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -84,6 +86,12 @@ trait ClusterComponent extends LeoComponent {
       }
     }
 
+    def getInitBucket(project: GoogleProject, name: String): DBIO[Option[GcsPath]] = {
```

Comment:
I don't think you're filtering on `project` and `name` in this method -- it's just returning the first cluster it finds.

Reply:
+1

Reply:
headdesk